packages/nest/README.md

```diff
@@ -0,0 +1,58 @@
+# NestJS Module Wrapper
+
+A NestJS module wrapper for [@notifire/core](https://github.com/notifirehq/notifire)
+
+## Usage
+
+Initializing module with templates and providers:
+```javascript
+    import { NotifireModule } from "@notifire/nest";
+
+    @Module({
+      imports: [
+        NotifireModule.forRoot({
+          providers: [
+            new SendgridEmailProvider({
+              apiKey: process.env.SENDGRID_API_KEY,
+              from: 'sender@mail.com',
+            }),
+          ],
+          templates: [
```

Comment:
Is that possible to declare providers' & templates separately ? 
for example service constructor ?

Reply:
@evgenyidf yeah sure, you can just use the NotifireService and call `regiserProvider` or `regiserTemplate` as usual.